tests/core/pyspec/eth2spec/test/altair/block_processing/test_process_sync_aggregate_random.py

```diff
@@ -0,0 +1,165 @@
+import random
```

Comment:
We might  consider putting both sync aggregate test files into a sub-directory `altair/block_processing/sync_aggregates/test_process_*.py`. I think this pattern would be more managable in the event of doing that type of sub-division in a phase with lots of tests. It's not critical here, but maybe marginally better... Thoughts?